### PR TITLE
179 xml tag handling

### PR DESCRIPTION
**Pull request by ShanaLMoore**
# Story

When I XML import an issue date, its year should appear as date accepted on the work's show page. 

This field should also be editable. 

Refs https://github.com/scientist-softserv/britishlibrary/issues/179
# Expected Behavior Before Changes

Before, the year would should up as 01/01/2016 (for example)

# Expected Behavior After Changes

Now, only the year should be displayed. 

# Screenshots / Video
Screenshot 2023-03-20 at 12 30 06 PM

Screenshot 2023-03-20 at 12 30 23 PM

# Notes

### CODE REVIEW

**Review by kirkkwang — APPROVED** [review]
Looking great!

File: app/presenters/hyku/work_show_presenter.rb [hunk 1]
```diff
@@ -45,7 +45,8 @@ def date_published
     def date_accepted
       date = solr_document["date_accepted_dtsim"]
       return formatted_date(date) if date.present?
-      solr_document["date_accepted_tesim"]
+      return if solr_document["date_accepted_tesim"].blank?
```
Comment (ShanaLMoore) [inline app/presenters/hyku/work_show_presenter.rb:48]: should we return an empty array if it's blank?

File: app/presenters/hyku/work_show_presenter.rb [hunk 2]
```diff
@@ -45,7 +45,8 @@ def date_published
...
+      solr_document["date_accepted_tesim"].map { |d| d.to_date.strftime("%Y") }
```
Comment (ShanaLMoore) [inline app/presenters/hyku/work_show_presenter.rb:49]: should [these other date methods](https://github.com/scientist-softserv/britishlibrary/blob/main/app/presenters/hyku/work_show_presenter.rb#L39-L55) receive the same treatment?

Comment (ShanaLMoore) [reply app/presenters/hyku/work_show_presenter.rb:49]: disregard, we should let the client dictate that.

File: app/presenters/hyku/work_show_presenter.rb (hunk 1 above)
Comment (ShanaLMoore) [reply app/presenters/hyku/work_show_presenter.rb:48]: tested and it just won't render anything at all, which is fine

**Pull request merged**